### ISSUE TITLE
Date selector dialog on restore option partially hidden

### ISSUE THREAD

**Issue by moorcito** [syncthing/syncthing#4755]
**- What happened**

Clicked on **Versions** > **Filter** by date and tried to set a time range.  This is what I saw.

![screenshot from 2018-02-13 10-59-32](https://user-images.githubusercontent.com/14274506/36144057-27b42c28-10ad-11e8-908e-fbffb63778d0.png)

The top of the dialog box is obscured by the address bar. This also happens in chromium.

**- What operating system, operating system version and version of Syncthing you are running**

Syncthing v0.14.45-rc.1, FreeBSD (64 bit) 
Firefox 58.0.2 (64-bit)
Chromium Version 64.0.3282.140 (Official Build) Built on Ubuntu , running on Ubuntu 17.10 (64-bit)

**Comment by AudriusButkevicius** [comment]
Yeah, it's not a mobile friendly UI, but there is nothing we can do sadly, uness some CSS guru wants to step up and fix this.

**Comment by uok** [comment]
workaround was for me: expand the folder/file tree which gives the page some vertical content and then select filter

**Comment by isavcic** [comment]
Still an issue. OS X Catalina on Retina, in Firefox and Safari both.

**Comment by AudriusButkevicius** [comment]
Yes, this issue is still open.
